### PR TITLE
Added more aliases

### PR DESCRIPTION
**Pull request by ghost**
Added aliases to
 1. Discard all changes
 2. Rebase
 3. Set Remote

### CODE REVIEW

**Comment by SarathSantoshDamaraju** [comment]
@grv19 
Can you change `gdsc`  to `g!.` Or `!g.` ?
I already have `g.` to stage all files and to discard all staged files, command should also be in the same pattern.

And congrats for your first PR 🎉

File: README.md [hunk 1]
```diff
@@ -46,6 +47,8 @@ Tired of typing lengthy git commands? Well, if you are amongst who feel like me,
 | `git stash drop [<stash>]`| `gsh-d [<stash>]`| Yes | [:information_source:](https://git-scm.com/docs/git-stash#git-stash-apply--index-q--quietltstashgt) |
 | `git tag -l --sort=v:refname \| tail -n8` | `gt` | No | [:information_source:](https://git-scm.com/docs/git-tag) |
 | `git add . && git commit -m` | `g.c <message>` | No, it accepts only commit message |[:information_source:](https://git-scm.com/docs/git-commit) |
+| `git rebase` | `grb <branch name>` | No| [:information_source:](https://git-scm.com/docs/git-rebase) |
```
Comment (SarathSantoshDamaraju) [inline README.md]: Your `git rebase` alias accepts default parameters, please update them as well.

File: README.md [hunk 2]
```diff
@@ -46,6 +47,8 @@ Tired of typing lengthy git commands? Well, if you are amongst who feel like me,
...
+| `git remote set-url origin` | `gremote <repourl>` | No| [:information_source:](https://git-scm.com/docs/git-remote) |
```
Comment (SarathSantoshDamaraju) [inline README.md]: Can you remove `gremote` as there is an alias already for it and it accepts all the default parameters with it.

**Comment by Gaurav-Shankar** [comment]
@SarathSantoshDamaraju  Good for merging now?

**Comment by ghost** [comment]
@SarathSantoshDamaraju  Good for merging now?

**Comment by SarathSantoshDamaraju** [comment]
Thanks @grv19

**Comment by grv19** [comment]
that's master branch. :P
lol

**Comment by SarathSantoshDamaraju** [comment]
Thanks @grv19

**Pull request merged**